<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReviewPage</class>
 <widget class="QWidget" name="ReviewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>680</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="rootLayout">
   <property name="leftMargin">
    <number>18</number>
   </property>
   <property name="topMargin">
    <number>16</number>
   </property>
   <property name="rightMargin">
    <number>18</number>
   </property>
   <property name="bottomMargin">
    <number>18</number>
   </property>
   <property name="spacing">
    <number>14</number>
   </property>

   <item>
    <layout class="QHBoxLayout" name="topBar">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QToolButton" name="btnBack">
       <property name="text">
        <string>←</string>
       </property>
       <property name="toolTip">
        <string>Назад</string>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblTitle">
       <property name="text">
        <string>Повторение</string>
       </property>
       <property name="heading" stdset="0">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="topSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>

   <item>
    <widget class="QStackedWidget" name="stack">
     <widget class="QWidget" name="pageEmpty">
      <layout class="QVBoxLayout" name="emptyLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <spacer name="emptyTopSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lblEmptyHint">
         <property name="text">
          <string>Выберите папку в библиотеке и нажмите «Повторять».</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignHCenter|Qt::AlignVCenter</set>
         </property>
         <property name="muted" stdset="0">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="emptyBottomSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>

     <widget class="QWidget" name="pageCard">
      <layout class="QVBoxLayout" name="cardLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="cardHost">
         <layout class="QVBoxLayout" name="cardHostLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>

  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
